src/main/java/org/cactoos/scalar/Repeated.java

```diff
@@ -23,48 +23,40 @@
  */
 package org.cactoos.scalar;
 
-import java.util.concurrent.Callable;
+import org.cactoos.Func;
+import org.cactoos.Scalar;
 import org.cactoos.func.FuncOf;
-import org.cactoos.func.Repeated;
 
 /**
- * Callable that runs repeatedly for a number of times.
+ * Scalar that runs repeatedly for a number of times.
  *
- * @param <X> Type of output
+ * @param <T> Type of output
  * @since 0.49.2
  */
-public final class RepeatedCallable<X> implements Callable<X> {
-
+public final class Repeated<T> implements Scalar<T> {
```

Comment:
@fabriciofx Not realated to the review. But what is the use case for such class? 
It seems that it applies some side effects, which is bad for `Scalar` to have
For example: 
`new Repeated<>(new Count<>(1), 5)).value() == 6 (all other values lost)`

Reply:
@andreoss Sorry, but I didn't get the point. I did some code here to test and works fine (but I can't make `Count<T>` because, I can't increment something like `T`:
```
@Test
void countFiveTimes() throws Exception {
    new Assertion<>(
        "Must repeat count 5 times",
        new Repeated<>(
            new Count(1),
            5
        ),
        new ScalarHasValue<>(5)
    );
}

final class Count implements Scalar<Integer> {
    private AtomicInteger num;

    public Count(final int num) {
        this.num = new AtomicInteger(num);
    }

    @Override
    public Integer value() throws Exception {
        return this.num.incrementAndGet();
    }
}
```

Reply:
@fabriciofx I mean, that with `Repeat` only the last result is returned, and all of other iteratatios are lost. Which implies mutable state. 
So `Repeated` may be seen as an encouragement to put mutable state in a `Scalar`, as in your `Count` example.